test/tables/gasp.mjs

```diff
@@ -0,0 +1,32 @@
+import assert from 'assert';
```

Comment:
bad rebase: this file is gasp.spec.mjs now

Reply:
OK, will fix.